third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/services/vertexai/resource_vertex_ai_endpoint.go

```diff
@@ -97,7 +98,7 @@ func ResourceVertexAIEndpoint() *schema.Resource {
 				Type:        schema.TypeString,
 				Optional:    true,
 				ForceNew:    true,
-				Description: `The full name of the Google Compute Engine [network](https://cloud.google.com//compute/docs/networks-and-firewalls#networks) to which the Endpoint should be peered. Private services access must already be configured for the network. If left unspecified, the Endpoint is not peered with any network. Only one of the fields, network or enable_private_service_connect, can be set. [Format](https://cloud.google.com/compute/docs/reference/rest/v1/networks/insert): 'projects/{project}/global/networks/{network}'. Where '{project}' is a project number, as in '12345', and '{network}' is network name.`,
+				Description: `The full name of the Google Compute Engine [network](https://cloud.google.com//compute/docs/networks-and-firewalls#networks) to which the Endpoint should be peered. Private services access must already be configured for the network. If left unspecified, the Endpoint is not peered with any network. Only one of the fields, network or enable_private_service_connect, can be set. [Format](https://cloud.google.com/compute/docs/reference/rest/v1/networks/insert): 'projects/{project_id}/global/networks/{network}'.`,
 			},
 			"region": {
 				Type:        schema.TypeString,
```

Comment:
Nit: Maybe explicitly mark this field required?

Reply:
Sorry which field are you referring to?

Reply:
Gotcha. Those fields are marked as required in KCC due to our own schema generation logic: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/pkg/crd/crdgeneration/tf2crdgeneration.go#L289

In practice they are required in KCC, but TF can have the global `region` configured, so they are optional within the resource. https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_reference.html